src/styles/utils/_loops.scss

```
@@ -0,0 +1,36 @@
+$week-days :
+  "mon" ($marginsForWeekDays) * 0,
+  "tue" ($marginsForWeekDays) * 1,
+  "wed" ($marginsForWeekDays) * 2,
+  "thu" ($marginsForWeekDays) * 3,
+  "fri" ($marginsForWeekDays) * 4,
+  "sat" ($marginsForWeekDays) * 5,
+  "sun" ($marginsForWeekDays) * 6;
```

correlate with the next comment
```suggestion
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```